### DISCUSSION TITLE
Question about threshold

### DISCUSSION THREAD

**Discussion by SaraCooperQA** [FRSOURCE/cypress-plugin-visual-regression-diff#214] [Q&A]
How do you change the threshold value so any difference will cause a fail?

**Comment by FRSgit** [comment]
Hey @SaraCooperQA,

[As of version `3.2.11`](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/issues/213#issuecomment-1484122783) it is possible to set `maxDiffThreshold: 0` in the plugin configuration.

Does this answer your question?

**Reply by donwelsh** [reply]
Hey @FRSgit. I am running `3.3.10`. This is my test: 
```
it('visual test', () => {
  cy.visit('https://example.cypress.io/todo')
  cy.wait(2000)
  cy.matchImage({
    diffConfig: {
      maxDiffThreshold: 0,
    },
  })
})
```

If I have a tiny diff it says: Image diff factor (0%) is within boundaries of maximum threshold option 0.01 and the test passes. If I have a big diff I get: Image diff factor (0.017%) is bigger than maximum threshold option 0.01. and the test correctly fails. I tried changing the maxDiffThreshold to different values, but it never caught the tiny diff. So I am having the same issue as @SaraCooperQA I believe.

**Reply by FRSgit** [reply]
@donwelsh that issue was fixed in `4.0.0`
